backbone.js

```diff
@@ -308,6 +308,7 @@
   Events.once = function(name, callback, context) {
     // Map the event into a `{event: once}` object.
     var events = eventsApi(onceMap, {}, name, callback, _.bind(this.off, this));
+    if (typeof name === 'string' && context == null) callback = void 0;
```

Comment:
Will this work?

```js
return this.on(events, typeof name === 'string' ? void 0 : callback, context);
```

I believe the issue is because we're converting from the string API into the object API. When using the object API to begin with, it should handle correctly.